ljps-frontend/pages/learning-journey/view-skills/view-courses/index.js

```diff
@@ -18,8 +18,8 @@ export default function ViewCourses() {
   const [roleName, setRoleName] = useState("");
 
   useEffect(() => {
-    if (skillID && roleID) {
-      const url = `${process.env.NEXT_PUBLIC_BACKEND}/api/skills/${skillID}`
+    if (skills && roleID) {
+      const url = `${process.env.NEXT_PUBLIC_BACKEND}/api/skills/multiple/${skills}`
```

Comment:
Is this endpoint changed?

Reply:
Hi Khye Chun, there were no changes to the endpoint. I simply reverted the api call on the frontend, cheers!